biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceUtils.java

```diff
@@ -474,4 +474,17 @@ public static String toProvideCapability(Capability cap) throws Exception {
 		return result;
 	}
 
+	public static boolean isFragment(Resource resource) {
+		List<Capability> capabilities = resource.getCapabilities(IdentityNamespace.IDENTITY_NAMESPACE);
+
+		if ((capabilities != null) && (capabilities.size() == 1)
+				&& (capabilities.get(0).getAttributes().get(IdentityNamespace.CAPABILITY_TYPE_ATTRIBUTE).equals(
+						IdentityNamespace.TYPE_FRAGMENT))) {
```

Comment:
Isn't this simply return the test in the if? That is `if (something) return true; else return false;` should just be `return something;`